Example/Example/Common/BankCard/BankCardViewController.swift

```diff
@@ -14,6 +14,50 @@
 
 import TextInputKit
 
+struct BankCard {
+
+    var cardNumber: BankCardNumber? = nil
```

Comment:
Yes. There was a `class` before. But `NSTextField.objectValue` expects an object to conform to `NSCopying`. I didn't want to subclass from `NSObject`, and changing `class` to `struct` fixed everything magically 🎁